tests/data/test_metapath_walker.py

```diff
@@ -317,11 +317,11 @@ def test_init_parameters(self):
         assert all(np.array_equal(w1, w2) for w1, w2 in zip(run_1, run_2))
 
     def test_benchmark_uniformrandommetapathwalk(self, benchmark):
-        g = example_graph_random(n_nodes=50, n_edges=500, node_types=2)
+        g = example_graph_random(n_nodes=50, n_edges=2000, node_types=4)
```

Comment:
Out of curiosity, why did this need to become larger? The metapaths got truncated often in practice, despite the theory?

Reply:
I tried increasing it since I wasn't so convinced that the speedups reported from the benchmark were as good as what we end up seeing in practice (https://github.com/stellargraph/stellargraph/pull/1399#issuecomment-623220728 vs https://github.com/stellargraph/stellargraph/pull/1399#issuecomment-624977346 the larger benchmark reports much higher speedup)

Reply:
Is that the number of edges increasing or the number of head nodes increasing?